code/workspaces/auth-service/src/permissions/__snapshots__/permissions.spec.js.snap

```diff
@@ -19,21 +19,30 @@ Array [
   "firstProjectName:permissions:read",
   "firstProjectName:permissions:create",
   "firstProjectName:permissions:delete",
+  "firstProjectName:projects:edit",
+  "firstProjectName:projects:read",
   "secondProjectName:stacks:create",
```

Comment:
I know you've not changed permissions.spec.js, but I think these should be firstProjectKey... not firstProjectName... in there.

Reply:
Good spot. I have updated these.